content/id/docs/concepts/architecture/control-plane-node-communication.md

```diff
@@ -15,31 +15,24 @@ Hal ini cukup penting, karena klaster bisa saja berjalan pada jaringan tak terpe
 
 <!-- body -->
 
-## Klaster menuju Master
+## Node Menuju Control Plane
 
-Semua jalur komunikasi dari klaster menuju master diterminasi pada apiserver.
-Tidak ada komponen apapun di dalam master, selain apiserver, yang terekspos ke luar untuk diakses dari servis <i>remote</i>.
-Untuk instalasi klaster pada umumnya, apiserver diatur untuk <i>listen</i> ke koneksi <i>remote</i> melalui port HTTPS (443) yang aman, dengan satu atau beberapa metode [autentikasi](/docs/reference/access-authn-authz/authentication/) <i>client</i> yang telah terpasang.
+Kubernetes memiliki sebuah pola API "hub-and-spoke". Semua penggunaan API dar node (atau pod dimana pod-pod tersebut dijalankan) akan diterminasi pada apiserver (tidak ada satu komponen control plane apa pun yang didesain untuk diekspos pada servis _remote_).
+Apiserver dikonfigurasi untuk mendengarkan koneksi aman _remote_ yang pada umumnya terdapat pada port HTTPS (443) dengan satu atau lebih bentuk [otentikasi](/docs/reference/access-authn-authz/authentication/) klien yang dipasang.
```

Comment:
nit: use https://kbbi.web.id/porta
also, autentikasi is more formal https://kbbi.kemdikbud.go.id/entri/autentikasi
```suggestion
Apiserver dikonfigurasi untuk mendengarkan koneksi aman _remote_ yang pada umumnya terdapat pada porta HTTPS (443) dengan satu atau lebih bentuk [autentikasi](/docs/reference/access-authn-authz/authentication/) klien yang dipasang.
```